modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/display/context/DLUIItemKeys.java

```diff
@@ -43,6 +43,9 @@
 
 	public static final String EDIT = DLUIItemKeys.class.getName() + "#edit";
 
+	public static final String EDIT_IN_GOOGLE =
```

Comment:
Hi @AliciaGarciaGarcia,

I don't think it is a good idea to add this to the global DL API module. It is specific to the google module.

What do you think about declaring that constant inside the `document-library-opener-google-drive-api` and adding a dependency to it in the google shortcut module?

Reply:
I didn't want to add another module dependency, and OPEN_IN_MS_OFFICE is on the same file.
So thats why I put it on there.
And there is the GoogleDocsUIItemKeys on document-library-google-docs so it's add the dependency to the google docs or change it

what do you think?

Reply:
I think it is better to add a dependency, because in reality you are creating a dependency anyway, and the key belongs to the other google module. 

Adding it to DL API means it will always be there even if one day we remove (or merge) the google modules. 

The MS Office one is probably there because there is not a module for that and it is from a very long time ago.